starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,90 @@
+body, html, main {
+  margin: 0;
+}
+
+html {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px
+}
+
+body{
+  font-weight: 100;
+}
+
+h1{
+  font-size: 45px;
+  font-weight: 100;
+}
+
+/* http://jsfiddle.net/qNjSp/ */
+ol {
+  counter-reset: list;
+}
+
+ol > li {
+  list-style: none;
+}
+
+ol > li:before {
+  content: counter(list) ")  ";
+  counter-increment: list;
+}
+/* http://jsfiddle.net/qNjSp/ */
+
+.hero{
+  height: 500px;
+  width: 100%;
+  position: relative;
+  text-align: center;
+}
+
+.hero-image{
+  background-image: url("../images/apple-pie.jpg");
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  height: 100%;
+
+}
+
+.apple-pie-heading{
+  color: white;
+  position: absolute;
+  top: 35%;
+  left: 40%;
+}
```

Comment:
Try not using position: absolute. That will give you issues especially when making your site responsive. You can also adjust the size of the font using font-size and the thickness of the font with font-weight.